src/isIsogram.test.js

```diff
@@ -6,4 +6,20 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return true for an empty string`, () => {
+    expect(isIsogram('')).toBe(true);
+  });
```

Comment:
Add more tests:
1. what should be returned when input is 1 letter?
2. what about Cyrillic letters?